actors/runtime/indices/indices.go

```diff
@@ -237,12 +237,15 @@ func (inds *IndicesImpl) StoragePower_PledgeSlashForSurprisePoStFailure(
 	panic("")
 }
 
+// FIL deposit per sector precommit in Interactive PoRep
+// refunded after ProveCommit but burned if PreCommit expires
 func (inds *IndicesImpl) StorageMining_PreCommitDeposit(
 	sectorSize abi.SectorSize,
 	expirationEpoch abi.ChainEpoch,
 ) abi.TokenAmount {
 	PARAM_FINISH()
-	panic("")
+	const PRECOMMIT_DEPOSIT_PER_BYTE = abi.TokenAmount(0) // placeholder
+	return abi.TokenAmount(int64(PRECOMMIT_DEPOSIT_PER_BYTE) * int64(sectorSize))
```

Comment:
Does the type signature of this method imply that this will be a constant value, and *not* a function of money supply? 

Please ensure all the inputs expected to these functions are specified.

Reply:
will confirm these params in separate PRs cc @jzimmerman